src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASKS = 100;
+    public static String horizontal = ("____________________________________________________________\n");
+
+    public static void sayHello() {
+        System.out.println(horizontal + "Hello! I'm Duke\n" + "What can I do for you?\n" + horizontal);
+    }
+
+    public static void sayBye() {
+        System.out.println(horizontal + "Bye. Hope to see you again soon!\n" + horizontal);
+    }
+
+    public static void addTask(String line, Task[] tasks, int numberOfTasks) {
+        if (line.contains("todo")) {
```

Comment:
Perhaps you may wish to replace magic strings such as "todo" or "deadline" with named constants. This will help readers understand your code better. I noticed the same issue in several other places too.